app/browser/tabs.js

```diff
@@ -617,6 +618,41 @@ const api = {
       api.createTab(state, action)
     }
     return state
+  },
+
+  goBack: (state, action) => {
+    action = makeImmutable(action)
+    const tab = api.getWebContents(action.get('tabId'))
+    tab.goBack()
```

Comment:
please wrap this with   `if (tab && !tab.isDestroyed()) {` and all below as well.